lightspark-sdk/src/commonTest/kotlin/com/lightspark/sdk/ClientIntegrationTests.kt

```diff
@@ -309,6 +309,46 @@ class ClientIntegrationTests {
         payments[0].id.shouldBe(payment?.id)
     }
 
+    // todo add payment has tests
```

Comment:
What do you mean by this one? Did you mean to remove this todo?

Reply:
ah this was junk code from trying out graphite, removing